src/query/storages/index/src/range_filter.rs

```diff
@@ -288,10 +288,22 @@ impl StatColumn {
                 single_point = false;
             }
 
-            let min_col = v.data_type().create_constant_column(&stat.min, 1)?;
+            let min = if let DataValue::Float64(f64::NAN) = stat.min {
```

Comment:
Add a stateless test for it?

Reply:
I just think about how to create `nan` index in stateless test, currently no idea.